### DISCUSSION TITLE
SD 1.5 or XL in Base Model

### DISCUSSION THREAD

**Discussion by kyoushinosensei** [lllyasviel/Fooocus#3016] [Q&A] [answered]
Hi.)) Please tell me - why can’t select SD 1.5 by default Base Model instead of XL, but only additionally? Is it possible to somehow get around this by setting, for example, some parameters in config..., or should we expect that there will be an update that will provide this opportunity?

**Comment by mashb1t** [comment, marked as answer]
Fooocus is a SDXL-first tool and SD1.5/2.1 support is not planned for base images. You can switch refiner after 0.1 (10%) to get an image mostly created by the refiner, which is not necessarily SDXL.

**Reply by Kh4rj1** [reply]
Could you explain why it is an SDXL first tool? Does the creator "want" us to use something different for SD1.5? I'm asking this from a very amateur perspective, imo Fooocus is just so much more intuitive than the alternatives and I'd love to be able to use it for SD1.5 as well.
Thanks!

**Reply by IPv6** [reply]
> Could you explain why it is an SDXL first tool?

Fooocus is free, purely open source and it is up to authors to limit amount of time and work they wish to dedicate to such development. SDXL was the deliberate choice for such limit - it is generally works better

SDXL and SD1.5 have some similarities, but a lot of differences too, so supporting both (with ALL features of Fooocus) means writing Fooocus almost twice and spending twice as time for supporting it further, afaik

**Reply by mashb1t** [reply]
^ this, thanks @IPv6
